webrtc.html

```diff
@@ -399,23 +399,31 @@
 
         <ol>
           <li>
-            <p>Append an object representing the current call being handled
-            (i.e. function name and corresponding arguments) to the
-            <code>operations</code> array.</p>
+            <p>Let <var>p</var> be the result of transforming
```

Comment:
The head of that document is "a collection of interesting ideas". How stable is it going to be?

Personally, I find "call p.then(fulfilment handler, undefined)" *MUCH* more readable than "transforming x by a fulfilment handler". Is there *any* reason why we should prefer the more convoluted langauge?